pkg/addons/data/Deployment.apps/openshift-monitoring/cluster-monitoring-operator.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: cluster-monitoring-operator
+  name: cluster-monitoring-operator
+  namespace: openshift-monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cluster-monitoring-operator
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: cluster-monitoring-operator
+    spec:
+      containers:
+      - args:
+        - -namespace=openshift-monitoring
+        - -configmap=cluster-monitoring-config
+        - -logtostderr=true
+        - -v=4
+        - -tags=prometheus-operator='v3.11.16'
+        - -tags=prometheus-config-reloader=v3.11.16
+        - -tags=config-reloader=v3.11.16
+        - -tags=prometheus=v3.11.16
+        - -tags=alertmanager=v3.11.16
+        - -tags=grafana=v3.11.16
+        - -tags=oauth-proxy=v3.11.16
+        - -tags=node-exporter=v3.11.16
+        - -tags=kube-state-metrics=v3.11.16
+        - -tags=kube-rbac-proxy=v3.11.16
```

Comment:
Any advise how we could parameterize this one? The derivative is not nice, stinks little bit. And its a list so jsonpath selector is not effective.

Reply:
More I look to this more I think it should be a config map not a flags.

Reply:
separate function to replace all block